content/syllabuses/bootcamp-data-eng-auto-boot.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Data engineering boot camp (with automarked katas)
+---
+
+- {{% contentlink path="projects/writing-assignment" flavour="markdown" %}}
+- {{% contentlink path="coding_aptitude_assessment/coding_challenges/task_1" flavour="python" %}}
+- {{% contentlink path="coding_aptitude_assessment/coding_challenges/task_2" flavour="python" %}}
+- {{% contentlink path="coding_aptitude_assessment/coding_challenges/task_3" flavour="python" %}}
+- {{% contentlink path="coding_aptitude_assessment/coding_challenges/task_4" flavour="python" %}}
+- {{% contentlink path="coding_aptitude_assessment/coding_challenges/task_5" flavour="python" %}}
+- {{% contentlink path="coding_aptitude_assessment/coding_challenges/task_6" flavour="python" %}}
+- {{% contentlink path="coding_aptitude_assessment/coding_challenges/task_7" flavour="python" %}}
+- {{% contentlink path="coding_aptitude_assessment/coding_challenges/task_8" flavour="python" %}}
+- {{% contentlink path="coding_aptitude_assessment/coding_challenges/task_9" flavour="python" %}}
+- {{% contentlink path="coding_aptitude_assessment/coding_challenges/task_10" flavour="python" %}}
+- {{% contentlink path="projects/git-exercises" %}}
+
+Things after this point are not strictly part of the bootcamp. It's extra stuff people can work on if they are ahead of the curve.
+
+- {{% contentlink path="topics/clean-code/python" %}}
+- {{% contentlink path="projects/tilde/repo-card-tutorial/part-2" %}}
+- {{% contentlink path="projects/katas/level-1" flavour="python" %}}
```

Comment:
I'm not sure that this stuff will work for the AWS folks. We don't want any humans to have to review any of their code at all.